lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -67,7 +67,7 @@ def pinning_override(self):
 
         ranktxt = ""
         sockets_per_node = 2
```

Comment:
this should be fixed but not in this PR: i made an issue https://github.com/hpcugent/vsc-mympirun/issues/97